voltixApp/VoltixApp/Services/Ethereum/EtherscanService.swift

```diff
@@ -174,6 +184,89 @@ public class EtherScanService: ObservableObject {
         }
     }
     
+    func fetchSafeFeeGwei() async throws -> Int64 {
```

Comment:
Can you please combine `fetchSafeFeeGwei` and `fetchPriorityFeeGwei` to return both base fee and priority fee , they both call the same API endpoint , the return result from api endpoint is
```
{
status: "1",
message: "OK",
result: {
LastBlock: "19521737",
SafeGasPrice: "29",
ProposeGasPrice: "29",
FastGasPrice: "29",
suggestBaseFee: "28.342071933",
gasUsedRatio: "0.2583948,0.546457566666667,0.3039359,0.631595633333333,0.504588566666667"
}
}
```
it contains all the information you need to figure out base fee and priority fee , so please make sure we only make one single api call , instead of two to get the same information

Reply:
Done

Reply:
Done